addusers.py

```diff
@@ -222,7 +266,11 @@ def parse_rows(rows, select_user=None):
             rows = select_rows(select_user, USER_COLUMN, rows)
         except ValueError as ve:
             raise argparse.ArgumentError(None, ve.message)
+    if select_test:
```

Comment:
These checks don't belong inside `parse_rows` at all.  You should be passing the rows created by `manual_testing` into this function.  They should be structured equivalently to the spreadsheet data.